python/ray/worker.py

```diff
@@ -710,9 +720,9 @@ def init(
             and the API is subject to change.
 
     Returns:
-        If the provided address included a protocol (e.g. "ray://1.2.3.4")
```

Comment:
Parentheses were throwing off sphinx doc generation (mixed up part inside the parens with return type)

Reply:
nice catch here!